.clang-tidy

```diff
@@ -14,6 +14,7 @@ Checks: >
   performance-*,
   -performance-avoid-endl,
   readability-*,
+  -readability-magic-numbers,
```

Comment:
To avoid the magic number warning for statements like [this](https://github.com/codeplaysoftware/portFFT/blob/cfd2ab8e9a3a662db941574d909e62117709c847/src/portfft/dispatcher/subgroup_dispatcher.hpp#L563), though avoiding the warning on a per line basis now